xml/admin_saltcluster.xml

```diff
@@ -372,6 +372,14 @@ block_wal_size: '5G'  # (optional, unit suffixes permitted)
 block_db_size: '5G'   # (optional, unit suffixes permitted)
 encrypted: true       # 'True' or 'False' (defaults to 'False')
 </screen>
+     <note>
+       <para>
+         When applying &drvgrps; in &productname; 7, the option previously 
+         called "Encryption" in &deepsea; has been renamed to "Encrypted".
```

Comment:
I think the option was previously called "encrypted" (not "Encrypted") and has been renamed to "encrypted" (not "Encrypted"), right?

Reply:
Indeed!